### PR TITLE
[libuv] upgrade to 1.6.1

### PR DESCRIPTION
**Pull request by saghul**
R=@misterdjules

### CODE REVIEW

**Comment by saghul** [comment]
Make that 1.6.1 :-)

File: deps/uv/src/win/fs-event.c [hunk 1]
```diff
@@ -225,11 +225,10 @@ int uv_fs_event_start(uv_fs_event_t* handle,
   }
 
   if (!handle->buffer) {
-    handle->buffer = (char*)_aligned_malloc(uv_directory_watcher_buffer_size,
-                                            sizeof(DWORD));
+    handle->buffer = (char*)uv__malloc(uv_directory_watcher_buffer_size);
```
Comment (misterdjules) [inline deps/uv/src/win/fs-event.c]: @saghul IIUC, here the call to `_aligned_malloc` would be replaced by a call to `malloc`, is it intentional and what is the impact of such a change?

**Comment by misterdjules** [comment]
@saghul [One question](https://github.com/joyent/node/pull/25475#discussion_r32255959) just to make sure I don't leave any stone unturned, otherwise that looks good. Will land asap unless the answer to that question means we need to change something.

Thank you once again @saghul :+1:

File: deps/uv/src/win/fs-event.c (hunk 1 above)
Comment (saghul) [inline deps/uv/src/win/fs-event.c]: That was reviewed back in the day and Bert pointed out that plain malloc
provided the necessary alignment in that case, that's why we dropped it.

Comment (misterdjules) [inline deps/uv/src/win/fs-event.c]: @saghul Sounds good, thank you for the clarification!

**Comment by misterdjules** [comment]
LGTM.

**Comment by misterdjules** [comment]
@saghul @joyent/node-collaborators Landing with http://jenkins.nodejs.org/job/node-accept-pull-request/127/.

**Comment by misterdjules** [comment]
Landed in 50f961596d5febb7c18dc01f28c257a423faf765 and 097607c782746ae0130f7e128ae44d5c4d146edc. Thank you very much once again @saghul :+1:

**Pull request closed without merging**